### DISCUSSION TITLE
schema + dimensions

### DISCUSSION THREAD

**Discussion by fatherfbi** [akvorado/akvorado#1570] [Q&A]
I added schema to akvorado.yaml
`schema:`
`  enabled:`
`    - SrcVlan`
`    - DstVlan`
SrcVlan and DstVlan are available in filters, but not in Dimensions
Can you tell me what the problem might be?

**Comment by vincentbernat** [comment]
Can you check if they are present with:

```
curl -s https://demo.akvorado.net/api/v0/console/configuration | jq .dimensions
```

(with the right URL for you)

**Reply by fatherfbi** [reply]
> Можете ли вы проверить их наличие с помощью:
> 
> ```
> curl -s https://demo.akvorado.net/api/v0/console/configuration | jq .dimensions
> ```
> 
> (с правильным для вас URL)

Thanks for the answer
After I executed this command, everything appeared
`root@akvodaro-dev:/akvorado# curl http://127.0.0.1:8080/api/v0/orchestrator/clickhouse/init.sh | sh`
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1949  100  1949    0     0   676k      0 --:--:-- --:--:-- --:--:--  951k
Install flow schema flow-LUZLB7VB35LILW22MLR2D27UM4.proto
Add Akvorado-specific configuration to ClickHouse
![2024-12-19_13-29-43](https://github.com/user-attachments/assets/c697dc6f-8ef7-4133-92c0-70c34e9b5963)

Your command returns this result
`root@akvodaro-dev:/akvorado# curl -s http://127.0.0.1:8080/api/v0/console/configuration | jq .dimensions
`
jq: error (at <stdin>:1): Cannot index number with string "dimensions"
jq: parse error: Invalid numeric literal at line 2, column 0
